capstone_project/src/main/java/com/google/sps/data/PostAnalysis.java

```diff
@@ -31,29 +33,50 @@
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.List;
+import java.util.HashMap;
+import java.util.Map;
 import javax.servlet.http.HttpServletRequest;
 
+
+/**
+ * Analyzes user posts to determine category classifications, which are used to provide
+ * users with resources in response to the specific categories
+ */
 public class PostAnalysis {
   private DatastoreService datastore;
   private final String message;
-  private final List<String> categories;
-  private final List<String> resources;
-
-  public PostAnalysis(HttpServletRequest request) throws IOException {
+  private final List <String> categories;
+  private final List <String> resources;
+ 
+  public PostAnalysis (HttpServletRequest request) throws IOException {
     datastore = DatastoreServiceFactory.getDatastoreService();
     categories = new ArrayList<>();
     resources = new ArrayList<>();
     message = classifyContent(request);
   }
 
+  /** Returns categories that the message falls under
```

Comment:
Add a newline after ```/**``` so "Returns categories..." is on the next line. Same for the other block comments.

Reply:
If you look at the output of "Java Presubmit / format (pull_request)" the parts with > before it are what it should look like